src/components/pages/tutorials/TutorialsContainer.js

```diff
@@ -6,11 +6,17 @@ import Tutorials from './Tutorials'
 import { updateCurrentUser } from '../../../redux/actions/currentUser'
 import selectIsFeatureDisabled from '../../router/selectors/selectIsFeatureDisabled'
 import { FEATURES } from '../../router/selectors/features'
+import { selectCurrentUser } from '../../../redux/selectors/currentUserSelector'
 
-export const mapStateToProps = state => ({
-  isHomepageDisabled: selectIsFeatureDisabled(state, FEATURES.HOMEPAGE),
-  isNewBookingLimitsActived: !selectIsFeatureDisabled(state, FEATURES.APPLY_BOOKING_LIMITS_V2),
-})
+export const mapStateToProps = state => {
+  const user = selectCurrentUser(state)
+  const isNewBookingLimitsActivated = user.deposit_version === null || user.deposit_version === 2
```

Comment:
Le nom de la variable est un peu piégeux : dans le composant qui l'utilise, on a l'impression qu'on dépend du *feature flag*.

Suggestion : 
```suggestion
  const userHasNewDepositVersion = user.deposit_version === null || user.deposit_version === 2
```